### ISSUE TITLE
finetune zipformer

### ISSUE THREAD

**Issue by 1215thebqtic** [k2-fsa/icefall#1312]
Hi,

I have a zipformer2 pretrained model. I used 1400h (800h from a business senario + 600h randomly selected from the data used to train the pretrained model) to finetune it. Although CER decreases 4% relatively on the specific senario, CER increases 15% on a general test set. It degrades too much on the general test set. 

I'd like to freeze some modules to mitigate the degradation, Can you give me some ideas on what modules I should freeze or other methods to mitigate the degradation?

Thanks!

**Comment by marcoyang1998** [comment]
I don't think freezing modules will help so much on the general domain. I would recommend you use more data from the original domain. Please have a look at the results shown in #1059.
